ci-operator/step-registry/workers-rhel/pre-hook/beta-repos/workers-rhel-pre-hook-beta-repos-commands.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+export OPS_MIRROR_KEY=${CLUSTER_PROFILE_DIR}/ops-mirror.pem
+
+echo "PLATFORM_VERSION: '${PLATFORM_VERSION}'"
+
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+oc version --client
+
+MIRROR_USERNAME="$(<'/var/run/mirror-repo-basic-auth/username')"
+export MIRROR_USERNAME
+MIRROR_PASSWORD="$(<'/var/run/mirror-repo-basic-auth/password')"
+export MIRROR_PASSWORD
+
+cat > prep-beta.yaml <<-'EOF'
+---
+- name: Prep Playbook
+  hosts: new_workers
+  any_errors_fatal: true
+  gather_facts: false
+
+  vars:
+    kubeconfig_path: "{{ lookup('env', 'KUBECONFIG') }}"
+    ops_mirror_path: "{{ lookup('env', 'OPS_MIRROR_KEY') }}"
+    platform_version: "{{ lookup('env', 'PLATFORM_VERSION') }}"
+    mirror_username: "{{ lookup('env', 'MIRROR_USERNAME') }}"
+    mirror_password: "{{ lookup('env', 'MIRROR_PASSWORD') }}"
+    major_platform_version: "{{ platform_version[:1] }}"
+
+  tasks:
+  - name: Get cluster version
+    command: >
+      oc get clusterversion
+      --kubeconfig={{ kubeconfig_path }}
+      --output=jsonpath='{.items[0].status.desired.version}'
+    delegate_to: localhost
+    register: oc_get
+    until:
+    - oc_get.stdout != ''
+
+  - name: Set release_version to cluster version
+    set_fact:
+      release_version: "{{ oc_get.stdout | regex_search('^\\d+\\.\\d+') }}"
```

Comment:
I guess we do not need `name: Set release_version to cluster version` and `name: Get cluster version` tasks.

Reply:
I'm not sure if we'll add rhel 7 beta in future, so leave a todo here 
`# NOTE: Add rhel 7 beta repo if needed`, if we have no plan to support rhel 7 beta, we can remove these.

Reply:
I guess we would never need to test rhel7 any more, based on we already have rhel8 coverage. Go through `ci-operator/step-registry/workers-rhel/repos/workers-rhel-repos-commands.sh`, `release_version` is only used to set repo name.